src/Compilers/CSharp/Portable/Symbols/VarianceSafety.cs

```diff
@@ -52,10 +54,70 @@ internal static void CheckInterfaceVarianceSafety(this NamedTypeSymbol interface
                     case SymbolKind.Event:
                         CheckEventVarianceSafety((EventSymbol)member, diagnostics);
                         break;
+                    case SymbolKind.NamedType:
+                        CheckNestedTypeVarianceSafety((NamedTypeSymbol)member, diagnostics);
```

Comment:
The intent here is to make any nested type inside a variant interface an error, right?

Reply:
> The intent here is to make any nested type inside a variant interface an error, right?

Not any nested type, delegates and interfaces are Ok.

---
In reply to: [347049921](https://github.com/dotnet/roslyn/pull/39839#discussion_r347049921) [](ancestors = 347049921)